.travis.yml

```diff
@@ -38,26 +38,26 @@ before_deploy:
   - pip install --user release-manager==0.2.0
 deploy:
   - provider: script
-    script: ./.travis/deploy_kinesis.sh $TRAVIS_TAG
+    script: ./.travis/deploy.sh kinesis $TRAVIS_TAG
     skip_cleanup: true
     on:
       tags: true
-      condition: '"$(.travis/is_kinesis_release_tag.sh $TRAVIS_TAG)" == "" && $? == 0'
+      condition: '"$(.travis/is_release_tag.sh kinesis $TRAVIS_TAG)" == "" && $? == 0'
   - provider: script
-    script: ./.travis/deploy_hadoop_enrich.sh $TRAVIS_TAG
+    script: ./.travis/deploy.sh spark_enrich $TRAVIS_TAG
     skip_cleanup: true
     on:
       tags: true
-      condition: '"$(.travis/is_hadoop_enrich_release_tag.sh $TRAVIS_TAG)" == "" && $? == 0'
+      condition: '"$(.travis/is_release_tag.sh spark_enrich $TRAVIS_TAG)" == "" && $? == 0'
   - provider: script
-    script: ./.travis/deploy_hadoop_shred.sh $TRAVIS_TAG
+    script: ./.travis/deploy.sh hadoop_shred $TRAVIS_TAG
     skip_cleanup: true
     on:
       tags: true
-      condition: '"$(.travis/is_hadoop_shred_release_tag.sh $TRAVIS_TAG)" == "" && $? == 0'
+      condition: '"$(.travis/is_release_tag.sh hadoop_shred $TRAVIS_TAG)" == "" && $? == 0'
   - provider: script
-    script: ./.travis/deploy_hadoop_enrich.sh $TRAVIS_TAG
```

Comment:
@alexanderdean Thanks, so this should have been deploy_hadoop_event_recovery.sh that's what I thought.